sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -266,18 +193,18 @@ export class EventProcessor {
   constructor(
     consumerGroupName: string,
     eventHubClient: EventHubClient,
-    partitionProcessorFactory: PartitionProcessorFactory,
+    PartitionProcessorClass: typeof PartitionProcessor,
```

Comment:
To make this stand out more, do you think `PartitionProcessorClassName` would be better?

Reply:
As long as no one tries to put in the class name as a string :) What about `PartitionProcessorConstructor`? Or is that as confusing?

Reply:
Meh, lets keep what you have for now.